client/lib/spine_client.dart

```diff
@@ -66,6 +66,10 @@
 ///                       ..who_completed = actorId;
 ///               client.post(requests.command().create(markDone));
 ///           });
+///     // Listen for all `TaskListView` updates and display them.
+///     Subscription<TaskListView> subscription =
+///             await client.subscribeTo(requests.topic().all(TaskListView()));
+///     subscription.itemChanged.listen(taskListView => updateDisplay(taskListView));
```

Comment:
Please place this above the querying. It's a common pattern to subscribe to updates and then query the current state. This way we do not lose any data.